src/gui/widgets/spinbox.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+	Copyright (C) 2008 - 2023
+	by Mark de Wever <koraq@xs4all.nl>
```

Comment:
By the way, unless this is you, it might be a good idea to delete this line from all files.

Reply:
Wasn't sure really if I should modify it. Thanks for pointing out.

Reply:
This line is different in different files and omitted in some of them, so I think it's indeed fine.